core/src/main/java/org/infinispan/xsite/CustomFailurePolicy.java

```diff
@@ -34,6 +36,10 @@
 
    void handleReplaceFailure(String site, K key, V oldValue, V newValue);
 
+   void handleComputeFailure(String site, K key, BiFunction<? super K, ? super V, ? extends V> remappingFunction, boolean computeIfPresent);
```

Comment:
While the `o.i.xsite` package is not marked as `@public` this interface is SPI, therefore by adding a non-default method you're breaking compatibility.